packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -100,6 +100,10 @@ export class GadgetConnection {
     this.baseClient = this.newBaseClient();
   }
 
+  private get sessionStorageKey() {
+    return `${sessionStorageKey}-${this.endpoint}`;
```

Comment:
This'll break anyone logged in with a current session when this ships for people. I think it wouldn't be too hard to check both the old deprecated key and also the new key for a little bit during the rollout, but only set it to the new key?

Reply:
Hm yea, I was thinking it's more convenient for anyone with one already stored at `token` to refetch a new session than having to understand that if there's one at `token` they'll get 401s until they clear localStorage when building embedded apps. Although that might be a smaller pool of users

Reply:
Talked IRL -- ok with logging people out in order to overcome having to clear localstorage for folks working on multiple apps locally.